### PR TITLE
Updated to include condition key for iam Passrole

### PR DESCRIPTION
**Pull request by ashishgore**
*Issue #, if available:*

*Description of changes:*
iam:PassRole with wildcard resource opens up to pass any role. Updated to scope only for MediaConnect.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/security_iam_resource-based-policy-examples.md [hunk 1]
```diff
@@ -41,7 +41,11 @@ To access the AWS Elemental MediaConnect console, you must have a policy that de
                 "iam:PassRole"
             ],
             "Effect": "Allow",
-            "Resource": "*"
+            "Resource": "*",
+            "Condition": {
+                "StringLike": {
+                    "iam:PassedToService": "mediaconnect.amazonaws.com"
```
Comment (serg-bern) [inline doc_source/security_iam_resource-based-policy-examples.md:47]: This block is missing the closed curly brackets

File: doc_source/security_iam_resource-based-policy-examples.md [hunk 2]
```diff
@@ -85,7 +89,11 @@ Every user of AWS Elemental MediaConnect must have a policy that defines permiss
...
```
Comment (serg-bern) [inline doc_source/security_iam_resource-based-policy-examples.md:96]: This block is missing the closed curly brackets

**Review by serg-bern — COMMENTED** [review]
This suggested change has been verified to work (though it needs the closing brackets in both the new `StringLike` blobs).

File: doc_source/security_iam_resource-based-policy-examples.md (hunk 1 above)
Comment (ashishgore) [reply doc_source/security_iam_resource-based-policy-examples.md:47]: Done. Validated the json this time. Sorry about that miss!

File: doc_source/security_iam_resource-based-policy-examples.md (hunk 2 above)
Comment (ashishgore) [reply doc_source/security_iam_resource-based-policy-examples.md:96]: Done. Validated the json this time. Sorry about that miss!

**Comment by kristywa** [comment]
Thank you so much for submitting this change! I really appreciate you taking the time to provide this suggestion. You'll see the change in the docs soon.

Thanks!
Kristy

**Pull request merged**